### PR TITLE
Reduce the need to access the disk in ES scripts

### PR DESCRIPTION
**Pull request by ross-spencer**
This commit removes the need to test the existence of an AIP on disk
when being indexed by our elasticsearch scripts by sending the
statistics that we are already in possession of when that script is
called. Additionally variables are cleaned up to perhaps be a bit
more intuitive to future readers of this code.

Connected to archivematica/issues#559
Connected to archivematica/issues#585
Cherry-picked from 1ec81430466688ae989f2413228d16d5849c9fee

### CODE REVIEW

**Review by jraddaoui — APPROVED** [review]
Thanks @ross-spencer!

I personally think there is no need to create a PR to cherry-pick a commit from the closest stable branch, unless there is a lot of conflicts in the process and you want an extra pair of eyes.

**Comment by ross-spencer** [comment]
Thanks @jraddaoui - I agree. I'll try the other way with the DSpace fix I've been working on. I have a feeling though our repository settings will block without approval though, so if you see a PR come through for that we'll know why :)

File: src/archivematicaCommon/lib/elasticSearchFunctions.py
Comment (alexwlchan) [inline src/archivematicaCommon/lib/elasticSearchFunctions.py]: Is this comment still correct? I don’t see a check for the existence of the AIP file.

Comment (ross-spencer) [reply src/archivematicaCommon/lib/elasticSearchFunctions.py]: Your're right. I missed that comment. I'll remove it before merging. Thanks @alexwlchan.

Comment (ross-spencer) [reply src/archivematicaCommon/lib/elasticSearchFunctions.py]: Fix it rather. The METS check is still there.

**Pull request merged**
